src/core/NotePlayHandle.cpp

```diff
@@ -252,8 +252,17 @@ void NotePlayHandle::play( sampleFrame * _working_buffer )
 	if( m_released && (!instrumentTrack()->isSustainPedalPressed() ||
 		m_releaseStarted) )
 	{
-		m_releaseStarted = true;
+		if (m_releaseStarted == false)
```

Comment:
`if (a)` is new convention, `if( a )` is old one. Many things are written with old convention, which make the code inconsistent for recently edited files.
I suggest using new style in line 252.